package.json

```diff
@@ -27,8 +30,11 @@
     "leaflet.markercluster": "^1.4.1",
     "main": "^1000.0.1",
     "main.js": "0.0.1",
+    "notistack": "^2.0.5",
     "papaparse": "^5.3.0",
     "pixelmatch": "^5.2.1",
+    "react": "^16.8.6",
```

Comment:
Is there a reason why we chose React 16 over React 18?

Suggest using 18.2.0 if possible.